GsaGH/Components/4_Analysis/EditAnalysisTask.cs

```diff
@@ -34,39 +34,39 @@ public class EditAnalysisTask : GH_OasysComponent
     #region Input and output
     protected override void RegisterInputParams(GH_InputParamManager pManager)
     {
-      pManager.AddParameter(new GsaAnalysisTaskParameter(), "Analysis Task", "ΣT", "GSA Analysis Task to Edit", GH_ParamAccess.item);
-      pManager.AddTextParameter("Name", "Na", "Set Task Name", GH_ParamAccess.item);
-      pManager.AddGenericParameter("Analysis Cases", "ΣAs", "Set List of GSA Analysis Cases", GH_ParamAccess.list);
-      pManager.AddTextParameter("Solver Type", "sT", "Set Solver Type" + System.Environment.NewLine +
-          "Default is: '1 : Static' - Accepted inputs are (either integer or text):" + System.Environment.NewLine +
-          "1 : Static" + System.Environment.NewLine +
-          "4 : Static_P_delta" + System.Environment.NewLine +
-          "8 : Nonlinear_static" + System.Environment.NewLine +
-          "2 : Modal_dynamic" + System.Environment.NewLine +
-          "5 : Modal_P_delta" + System.Environment.NewLine +
-          "32 : Ritz" + System.Environment.NewLine +
-          "33 : Ritz_P_Delta" + System.Environment.NewLine +
-          "6 : Response_spectrum" + System.Environment.NewLine +
-          "42 : Pseudo_Response_spectrum" + System.Environment.NewLine +
-          "15 : Linear_time_history" + System.Environment.NewLine +
-          "14 : Harmonic" + System.Environment.NewLine +
-          "34 : Footfall" + System.Environment.NewLine +
-          "35 : Periodic" + System.Environment.NewLine +
-          "3 : Buckling" + System.Environment.NewLine +
-          "9 : Form_finding" + System.Environment.NewLine +
-          "37 : Envelope" + System.Environment.NewLine +
-          "39 : Model_stability" + System.Environment.NewLine +
-          "40 : Model_stability_P_delta", GH_ParamAccess.item);
-      pManager.AddIntegerParameter("TaskID", "ID", "The Task number - only set this if you want to append cases to an existing task.", GH_ParamAccess.item);
+      pManager.AddParameter(new GsaAnalysisTaskParameter(), GsaAnalysisTaskGoo.Name, GsaAnalysisTaskGoo.NickName, GsaAnalysisTaskGoo.Name + " to Edit", GH_ParamAccess.item);
```

Comment:
Is this a temporary change? Or why commented out?

Reply:
Yes these aren't available to set/create in GsaAPI at the moment.